scripts/release_notes.rb

```diff
@@ -10,14 +10,19 @@
 
 URL = 'https://projects.theforeman.org'
 
-unless ARGV.size == 2
-  puts "Usage: #{$0} PROJECTS RELEASE_NAME"
+unless ARGV.size <= 2
+  puts "Usage: #{$0} PROJECTS RELEASE_NAME [markdown|asciidoc]"
```

Comment:
Should we print what is the default output mode or not required?

Reply:
Yup.